WatermelonDB.podspec

```diff
@@ -0,0 +1,19 @@
+require "json"
+
+  package = JSON.parse(File.read("package.json"))
+
+  Pod::Spec.new do |s|
+   s.name         = "WatermelonDB"
+   s.version      = package["version"]
+   s.summary      = package["description"]
+   s.description  = package["description"]
+   s.homepage     = package["homepage"]
+   s.license      = package["licence"]
+   s.author       = { "author" => package["author"] }
+   s.platforms    = { :ios => "9.0", :tvos => "9.0" }
+   s.source       = { :http => 'file:' + __dir__ + '/'  }
+   s.source_files = "native/ios/**/*.{h,m,swift}"
```

Comment:
i think FMDB/ needs to be skipped

Reply:
No need. But needs to remove the dependence below